pypeit/scripts/chk_edges.py

```diff
@@ -22,15 +23,12 @@ def parser(options=None):
     parser.add_argument('--try_old', default=False, action='store_true',
                         help='Attempt to load old datamodel versions.  A crash may ensue..')
 
+    if return_parser:
+        return parser
+
     return parser.parse_args() if options is None else parser.parse_args(options)
 
```

Comment:
I feel this TODO should remain. Either we need a separate check edges for the slit_trace object script, or the show function should be moved there.

Reply:
Added back.